core-requirements.txt

```diff
@@ -11,7 +11,7 @@ psutil>=5.6.6
 requirements-parser>=0.2.0
 shap>=0.36.0
 texttable>=1.6.2
-woodwork>=0.4.1
+woodwork>=0.4.1,<=0.4.2
```

Comment:
Should we do `woodwork>=0.4.1,<0.5.0`? If they follow semantic versioning, 0.5.0 will be the next breaking change

@gsheni

Reply:
Agreed, `<0.5.0` is the right approach, and will avoid the upcoming breaking change.